src/cwe_checker_lib/src/checkers/cwe_190.rs

```diff
@@ -97,6 +99,24 @@ fn generate_cwe_warning(callsite: &Tid, called_symbol: &ExternSymbol) -> CweWarn
         .symbols(vec![called_symbol.name.clone()])
 }
 
+/// Determines if the argument for the call contains relative intervals or top values for absolute intervals.
+fn contains_top_value(pir: &PointerInference, jmp_tid: &Tid, parms: Vec<&Arg>) -> bool {
+    for arg in parms {
+        let value = pir.eval_parameter_arg_at_call(jmp_tid, arg).unwrap();
+        dbg!(&value);
+        if let Some(inter_dom) = value.get_absolute_value() {
+            if inter_dom.get_interval().is_top() {
+                return true;
+            }
+        }
+        if !value.get_relative_values().is_empty() {
+            return true;
+        }
```

Comment:
You can use the method `value.get_if_absolute_value()` to get the contained absolute values and make sure that no other type of values are contained in the domain at the same time.

Also, the interval domain implements the `AbstractDomain` trait, i.e. you can simply write `inter_dom.is_top()` without unwrapping the contained interval first.